static/js/home/snatch-selection.js

```diff
@@ -130,36 +130,36 @@ MEDUSA.home.snatchSelection = function() {
             if (data.result === 'refresh') {
                 self.refreshResults();
                 updateSpinner(spinner, 'Refreshed results...', true);
-                initTableSorter('#resultTable');
+                initTableSorter('#srchresults');
             }
             if (data.result === 'searching') {
                 // ep is searched, you will get a results any minute now
                 pollInterval = 5000;
                 $('.manualSearchButton').prop('disabled', true);
                 updateSpinner(spinner, 'The episode is being searched, please wait......', true);
-                initTableSorter('#resultTable');
+                initTableSorter('#srchresults');
```

Comment:
Please don't use shortened words, I know this sounds silly to point out something so simple but it makes it harder to understand what it is later on and you're only saving 2 chars.